README.md

```diff
@@ -12,9 +12,10 @@
 Fabricator is a tool for building website UI toolkits - _think ["Tiny Bootstraps, for Every Client"](http://daverupert.com/2013/04/responsive-deliverables/#tiny-bootstraps-for-every-client)_
 
 ## Quick Start
-
+First we need to install the [Git](https://git-scm.com/downloads) and [Node.js](https://git-scm.com/downloads).
```

Comment:
I don't believe that git is an actual requirement.

Reply:
It will useful for Windows env, now the days its a basic requirement to install GIT.